.travis.yml

```diff
@@ -36,6 +37,7 @@ env:
    - CCACHE_VERSION=3.3.1
    - CMAKE_VERSION=3.7.2
    - MASON="$(pwd)/scripts/mason.sh"
+   - NODE="4"
```

Comment:
I don't understand how this interacts with the `node_js` values set above. Is this just to have a default, and will the values above overwrite node 4 here?